### DISCUSSION TITLE
Failed to build on http://railway.app

### DISCUSSION THREAD

**Discussion by magadan** [hertzg/tesseract-server#59] [General]
Failed to build on http://railway.app:

Step 21/32 : COPY $DIST_SRC/index.js $DIST_SRC/*.production.*.js ./dist/
COPY failed: file not found in build context or excluded by .dockerignore: stat index.js: file does not exist

**Comment by hertzg** [comment]
And? Do you want to support railway app? If so feel free to propose.

**Comment by magadan** [comment]
Hello George, yes, i thinking to install on it, i will install in the default way (standard). I will try to solve the copy problem. Thanks for the project very good.

**Comment by hertzg** [comment]
Let's continue discussion in the discussions section as this does not seem like an issue but more of a discussion. 👍
